docs/generated/cli/create-nx-workspace.md

```diff
@@ -69,7 +69,7 @@ Workspace name (e.g. org name)
 
 Type: boolean
 
-Default: true
+Default: "true"
```

Comment:
@bcabanes this one is tricky, it shouldn't have quotes and it wouldn't have it if the value is specified as a boolean, but it's specified as a string because it's the description of the default value [`defaultDescription: 'true'`](https://github.com/nrwl/nx/blob/master/packages/create-nx-workspace/bin/create-nx-workspace.ts#L157).

We could set `defaultDescription: true as any` and the docs would be generated correctly. On the other hand, I don't even think the `defaultDescription` should be set, because we always prompt for NxCloud (unless the arg is provided), therefore there's not really a default value, it's just the initial selected value in the prompt.

Anyways, didn't make any of those changes to keep this focused on solely docs changes.

Reply:
I'm not sure why the default value is provided as a `string` if it should be a `boolean`. Do you know? Is it because of Yargs?

Reply:
If it's specified as a `default`, the prompt won't be asked, because Yarn will set the default value automatically.

Reply:
Okay so we can do a really naive thing that should work: 
```javascript
JSON.stringify(e).replace(/"/ig, '')
```

Will solve the problem for now, we will refactor that part soon anyway.

Reply:
Yeah definitely, I did consider that but didn't do it because that would remove the quotes from string values which IMHO is also wrong. I guess folks would infer is a string value, but there's no visual cue to differentiate `main` from `true` when there are no quotes. Part of the reason for this change was to correct all the string values to have quotes, right now is a mix, some have it, and some don't.

If we still want to remove the quotes, then I'd remove all quotes to at least be consistent, but we should consider using the quotes for string values when you do refactor that part.

Reply:
That description is being removed in https://github.com/nrwl/nx/pull/9747. Once that PR is merged, I'll rebase this one which should remove this issue.

Reply:
The nx cloud false "default" value is removed here: https://github.com/nrwl/nx/pull/9747.
The two remaining `packageManager` and `defaultBase` should stay as string, but there is upcoming `ci` param which will have default value `[]`. Ideally that should be without quotes, but I guess leaving it with quotes will not make too much of an issue.

Reply:
@bcabanes I rebased this and removed the quotes from all values to avoid future issues with the `defaultDescription` property.